org.wownero.wownerod.json

```diff
@@ -0,0 +1,34 @@
+{
+  "app-id": "org.wownero.wownerod",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "/app/bin/wownerod",
+  "finish-args": [
+    "--filesystem=host",
+    "--share=network"
+  ],
+  "modules": [
+    {
+      "name": "wownero",
+      "buildsystem": "simple",
+      "build-commands": [
+        "ar x wownero_0.5.0.0_amd64.deb",
+        "tar -xf data.tar.xz",
+        "install -Dm755 usr/local/bin/wownerod /app/bin/wownerod",
+        "install -Dm644 org.wownero.wownerod.appdata.xml /app/share/appdata/org.wownero.wownerod.appdata.xml"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "url": "http://ppa.wownero.com/pool/main/w/wownero/wownero_0.5.0.0_amd64.deb",
```

Comment:
From what I see it's open source, so it would be preferred to build it instead of repackaging.